Author: Tim Burks <timburks@google.com>
Date: Tue Jul 14 13:24:18 2020 -0700
Forwarded: not-needed
Subject: Add new ParseDocument() functions to directly read OpenAPI v2 and v3 docs from bytes.

--- /dev/null
+++ b/OpenAPIv2/document.go
@@ -0,0 +1,26 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openapi_v2
+
+import "github.com/googleapis/gnostic/compiler"
+
+// ParseDocument reads an OpenAPI v2 description from a YAML/JSON representation.
+func ParseDocument(b []byte) (*Document, error) {
+	info, err := compiler.ReadInfoFromBytes("", b)
+	if err != nil {
+		return nil, err
+	}
+	return NewDocument(info, compiler.NewContextWithExtensions("$root", nil, nil))
+}
